mlrun/api/utils/scheduler.py

```diff
@@ -278,20 +306,67 @@ def _remove_schedule_secrets(
         import mlrun.api.crud
 
         if self._store_schedule_credentials_in_secrets:
-            # sanity
-            secret_key = mlrun.api.crud.Secrets().generate_schedule_secret_key(name)
+            access_key_secret_key = mlrun.api.crud.Secrets().generate_schedule_access_key_secret_key(
+                name
+            )
+            username_secret_key = mlrun.api.crud.Secrets().generate_schedule_username_secret_key(
+                name
+            )
             secret_key_map = (
                 mlrun.api.crud.Secrets().generate_schedule_key_map_secret_key()
             )
             mlrun.api.crud.Secrets().delete_secret(
```

Comment:
That's an optimization, I'm not sure it will be that easy to do that, anyways let's leave it to another PR
I already added a TODO for the list case (in `_get_schedule_secrets`), now I added one in `_remove_schedule_secrets` as well, for the deletion case